src/client/app/components/home/home.component.html

```diff
@@ -18,4 +18,6 @@
   <li *ngFor="let name of nameListService.names | async">{{name}}</li>
 </ul>
 
-<a href (click)="readAbout()">About</a>
+<p>
+  <a href (click)="readAbout($event)">About</a>
```

Comment:
Lastly, lets just give the link this text: `Navigate to About Example using RouterExtensions`  to make it clear on the web sample what that's for.